crates/compiler/test_gen/src/helpers/llvm.rs

```diff
@@ -611,8 +612,8 @@ where
             std::mem::forget(given);
         }
         Err((msg, tag)) => match tag {
-            CrashTag::Roc => panic!(r#"Roc failed with message: "{}""#, msg),
-            CrashTag::User => panic!(r#"User crash with message: "{}""#, msg),
```

Comment:
I think the test failures may be due to changing these lines. The way our internal tests work, I believe they require just a standard panic here.

Reply:
fixed in